website/contributing/release-updating-packages.md

```diff
@@ -0,0 +1,93 @@
+---
+id: release-updating-packages
+title: Updating packages
+---
+
+This page contains relevant information about how to update packages in the `react-native` [monorepo](https://github.com/react-native-community/discussions-and-proposals/pull/480).
+
+## Finding all packages that have unpublished changes
+
+#### Use case
+
+1. You want to identify each package with unpublished (on npm) changes and update its version. This can be used in release cycle if you have merged some fixes to `*-stable` branch.
+2. You want to force-bump each public package to the next minor version. This happens usually before release branch cutoff. In this case, please specify `release-branch-cutoff` argument before executing the script.
+
+#### How to execute
+
+`npm run bump-all-updated-packages` or `npm run bump-all-updated-packages --release-branch-cutoff`
```

Comment:
```suggestion
`npm run bump-all-updated-packages` or `npm run bump-all-updated-packages -- --release-branch-cutoff`
```

Reply:
I used `yarn` instead since that's the tool of choice for other commands such as running `test-e2e-local`

Reply:
> I used `yarn` instead since that's the tool of choice for other commands such as running `test-e2e-local`

Can you please update examples of executing this command with yarn? I don't have a strong opinion on if we should include both, seems pretty straightforward, but definitely better for it to be either `yarn` or `npm` everywhere

Reply:
Fair point, done 👍